src/styling-panel-definition.js

```diff
@@ -1,4 +1,86 @@
-const getStylingPanelDefinition = (bkgOptionsEnabled) => ({
+import styleDefaults from './style-defaults';
+import propertyResolver from './utils/property-resolver';
+import { fontFamilyOptions, colorOptions } from './utils/style-utils';
+
+export const getStyleEditorDefinition = () => ({
+  items: {
+    labelSection: {
+      component: 'panel-section',
+      translation: 'Common.Label',
+      items: {
+        labelItem: {
+          items: {
+            fontFamily: {
+              component: 'dropdown',
+              ref: 'style.font.fontFamily',
+              options: fontFamilyOptions,
```

Comment:
Would be great if we can have this in the client/analytics instead. Otherwise we need to keep a copy of the array in every chart that we want to use it. For now this is fine